tools/clang/lib/SPIRV/EmitSPIRVAction.cpp

```diff
@@ -802,6 +949,17 @@ case BO_##kind : {                                                             \
       return theBuilder.getConstantFloat32(1.0);
     }
 
+    if (hlsl::IsHLSLVecType(type)) {
+      const QualType elemType = hlsl::GetHLSLVecElementType(type);
+      const uint32_t elemOneId = getValueOne(elemType);
+
+      const size_t size = hlsl::GetHLSLVecSize(type);
+      llvm::SmallVector<uint32_t, 4> elements(size, elemOneId);
```

Comment:
Good catch! Actually this was fixed in a later CL: https://github.com/antiagainst/DirectXShaderCompiler/commit/58aef4471ca614a888df767b22893a6095591f4e#diff-d00a9d7516b840ab223da9e8420a9844R1181

I'll move that change forward to this CL.